src/App.jsx

```diff
@@ -1,22 +1,24 @@
-import './App.css';
-import Navbar from './components/navbar/Navbar';
-import { BrowserRouter as Router, Route, Routes } from 'react-router-dom';
-import Homepage from './components/homepage/Homepage';
-import Create from './components/homepage/Create';
-import About from './components/about/About';
+import "./App.css";
+import Navbar from "./components/navbar/Navbar";
+import { BrowserRouter as Router, Route, Routes } from "react-router-dom";
+import Homepage from "./components/homepage/Homepage";
+import Create from "./components/navbar/Create";
+import About from "./components/about/About";
+import MovieDetails from "./components/movie/MovieDetails";
 
 function App() {
   return (
     <Router>
       <div className="App">
         <Navbar />
         <Routes>
-          <Route path="" element={<Homepage />} />
-          <Route path="about" element={<About />} />
-          <Route path="create" element={<Create />} />
+          <Route exact path="" element={<Homepage />} />
```

Comment:
why did you add `exact` again?

Reply:
I reverted the commit from the last branch that is why it got again in this code. It is updated now.